ivy/src/main/scala/sbt/internal/librarymanagement/MakePom.scala

```diff
@@ -206,8 +206,13 @@ class MakePom(val log: Logger) {
             <developer>
               <id>{ developer.id }</id>
               <name>{ developer.name }</name>
-              <email>{ developer.email }</email>
               <url>{ developer.url }</url>
+              {
+                developer.email match {
```

Comment:
Maybe we can do something like if email is `""` make it `NodeSeq.Empty`?

Reply:
seems legit

Reply:
Should we also do same for null case?